provisioners/configure.sh

```diff
@@ -16,8 +16,14 @@ apt-get -qq update
 apt-get -qq install -y curl htop wget build-essential zip software-properties-common gnupg awscli
 
 echo $PERM_ENV  > /data/www/host.txt
+#preseeding NR
+echo newrelic-php5 newrelic-php5/application-name string $NEW_RELIC_APPLICATION_NAME | debconf-set-selections
+echo newrelic-php5 newrelic-php5/license-key string $NEW_RELIC_LICENSE_KEY | debconf-set-selections
```

Comment:
These preseeding lines look good, but the variables are not yet populated. You'll need to pipe the values from GitHub secrets through packer and ansible. See also [my comment about the Sentry values](https://github.com/PermanentOrg/infrastructure/pull/51#discussion_r610280627).